src/editor/mainEditor/material/components/ColorPicker.tsx

```diff
@@ -1,8 +1,7 @@
 import * as React from "react";
-import { connect } from "react-redux";
 import { SketchPicker, BlockPicker, ChromePicker, CirclePicker, CompactPicker, HuePicker, MaterialPicker, SliderPicker, SwatchesPicker } from 'react-color';
 import reactCSS from 'reactcss';
-import { string2rgb, reverseRGB, rgb2hex, hex2string } from '../../utils/colorUtil';
+import { string2rgb, reverseRGB, rgb2hex, hex2string } from '../../../../utils/colorUtil';
```

Comment:
refactor: material/components/ColorPicker.tsx should move to material/ui/components/

Reply:
resolved